common-lib/src/main/resources/application.conf

```diff
@@ -23,3 +23,22 @@ image.processors = [
   "com.gu.mediaservice.lib.cleanup.GuardianMetadataCleaners",
   "com.gu.mediaservice.lib.cleanup.SupplierProcessors$"
 ]
+
+authentication.providers {
+  machine {
+    className = "com.gu.mediaservice.lib.auth.provider.ApiKeyAuthenticationProvider"
+    config {
+      # authKeyStoreBucket = <s3-bucket-with-api-keys>
+    }
+  }
+  # TODO: short term we put panda here for backwards compatibility but the default provider should be something better
+  user {
+    className = "com.gu.mediaservice.lib.guardian.auth.PandaAuthenticationProvider"
+    config {
+      # all of the things relating to pan domain auth (these are currently sensibly defaulted in code)
```

Comment:
Worth saying something like "sensibly defaulted for Guardian usage"?

Reply:
I kind of figured that's implicit in that the provider is Guardian specific, but I will make that clearer.